DOKS-automatic-node-repair/README.md

```diff
@@ -1 +1,402 @@
-TBD
+# DigitalOcean Kubernetes Automatic Node Repair
```

Comment:
In this tutorial, we show how to use Digital Mobius to automatically replace unhealthy DigitalOcean Kubernetes (DOKS) cluster nodes automatically. DOKS cluster pool nodes can transition to an unhealthy state (`NotReady` or `Unknown`) by user will or by accident (overloading, deleting `kube-system` stuff, `kubelet` issues). Using an auto-repair mechanism replaces the affected nodes from the cluster pool automatically.